src/etc/inc/ipsec.inc

```diff
@@ -2013,9 +2017,13 @@ function ipsec_setup_tunnels() {
 						empty($p1enc['hash-algorithm'])) {
 					continue;
 				}
+				if ($ph1ent['prfselect_enable'] != 'yes') {
+					$p1enc['prf-algorithm'] = False;
```

Comment:
This should either be `false` or `FALSE`, preferably `false` since that's what is used elsewhere in the file.

Reply:
done